src/webrtc/opcodes/index.ts

```diff
@@ -25,7 +25,7 @@ import { onSelectProtocol } from "./SelectProtocol";
 import { onSpeaking } from "./Speaking";
 import { onVideo } from "./Video";
 
-export type OPCodeHandler = (this: WebSocket, data: Payload) => any;
+export type OPCodeHandler = (this: WebSocket, data: Payload) => unknown;
```

Comment:
why was this changed?

Reply:
because we have an eslint rule against using `any`.